directors/basic_director.py

```diff
@@ -109,7 +110,7 @@ def process_frame(self, frame : list):
                             print(rotation)
                             rotation = int(round(rotation))
                             #Publisher.rotate_altitude(rotation)
-                            Publisher.polar_pan(0, rotation, 0, 3000)
+                            #Publisher.polar_pan_discrete(0, rotation, 0, 3000)
```

Comment:
Should this be commented out?

Reply:
So when we were testing on Saturday it was commented out. I wasn't sure if it being commented out was part of the demo

Reply:
Just for the altitude not the azimuth